frb_codegen/src/generator/c/mod.rs

```diff
@@ -1,14 +1,84 @@
-pub fn generate_dummy(func_names: &[String]) -> String {
+use std::path::Path;
+
+use crate::config;
+use crate::utils::{relative_path, BlockIndex, PathExt};
+
+pub fn generate_dummy(
+    config: &config::Opts,
+    configs: &[config::Opts],
+    func_names: &[String],
+    c_path_index: usize,
+) -> String {
+    match configs.len() > 1 {
+        true => {
```

Comment:
uber-nit: if-else

Reply:
fixed in https://github.com/fzyzcjy/flutter_rust_bridge/pull/1024/commits/cbd8da41d3f47156452430b5a2774aa471e2ddb2.